crates/blockifier/src/concurrency/flow_test.rs

```diff
@@ -66,13 +70,14 @@ fn scheduler_flow_test(
                         versioned_state.pin_version(tx_index).apply_writes(
                             &writes,
                             &ContractClassMapping::default(),
-                            &HashMap::default(),
+                            &VisitedPcsSet::default(),
                         );
                         scheduler.finish_execution(tx_index);
                         Task::AskForTask
                     }
                     Task::ValidationTask(tx_index) => {
-                        let state_proxy = versioned_state.pin_version(tx_index);
+                        let state_proxy: VersionedStateProxy<_, VisitedPcsSet> =
```

Comment:
Is this type annotation needed?

Reply:
`VersionedStateProxy` requires the type annotation because it can't be inferred from the argument of `pin_version`.